### DISCUSSION TITLE
Is multiple output support on the roadmap?

### DISCUSSION THREAD

**Discussion by ghost** [mikebrady/shairport-sync#1487] [Ideas]
I've seen a few people asking about running multiple instances, clearly to try to support multiple audio outs from a device. Instead, are there any plans to expand the project to be able to define an array of sinks and outputs?

**Comment by mikebrady** [comment]
Shairport Sync can already do the multiple instances thing, but AirPlay 2 players seem to get confused by having multiple audio outs on the same IP numbers.

**Comment by noelhibbard** [comment]
ALSA can easily be configured to output to multiple outputs.
